app/services/db/monthly_sp_auth_count/unique_monthly_auth_counts_by_iaa.rb

```diff
@@ -116,7 +116,7 @@ def partial_month_subqueries(issuers:, partial_months:)
             , sp_return_logs.ial
             FROM sp_return_logs
             WHERE
-                  sp_return_logs.requested_at BETWEEN %{range_start} AND %{range_end}
+                  requested_at::date BETWEEN %{range_start} AND %{range_end}
```

Comment:
ditto for keeping the table name
```suggestion
                  sp_return_logs.requested_at::date BETWEEN %{range_start} AND %{range_end}
```